scripts/cover.sh

```diff
@@ -13,6 +13,7 @@ if [ "$id_new" != "$id_current" ]; then
 
 	    imgurl=`~/.conky/conky-spotify/scripts/imgurl.sh $id_new`
 	    wget -q -O ~/.conky/conky-spotify/covers/$id_new.jpg $imgurl &> /dev/null
+        touch $id_new.jpg
```

Comment:
Thanks for the contribution :smiley: 

However, I am not sure if this change really does that you want. This creates a new empty file in `~/.conky` directory when the cover changes:

![imagen](https://user-images.githubusercontent.com/5855179/109389491-c6f97780-7904-11eb-97f4-2904e5977df3.png)